fantasy_matchup.py

```diff
@@ -37,10 +37,10 @@ def __init__(self, owner, team, year, week, matchup):
 		self.team_name = team
 		self.week = week
 		self.year = year
-		if owner == matchup.home_team.owner.replace("  ", " ").strip().title():
+		if owner == matchup.home_team.owner:
```

Comment:
Do these need to have clean_name run on them?

Reply:
Yeah I'm actually hitting an issue I missed early related to these lines, kinda related anyway, I have to make an update to here

Reply:
Fixed a change I made further upstream (`fantasy_league.py` lines ~115-125) that was intended to make it so we don't need `clean_name` here.  So now we actually don't need it, nor do we need the `try/except`s here anymore

Reply:
Oh hell yeah!! Nice work!